doc/changelog_legend.inc

```diff
@@ -0,0 +1,12 @@
+Legend for changelogs
```

Comment:
Why not put this file in the whats_new folder?

Reply:
Fixed... definitely cleaner, thanks.